### PR TITLE
Change window size and enemy location, Update documentation.

### PR DESCRIPTION
**Pull request by kylegessel**
This changes the window size and puts the enemies in one spot as a QOL change. I would like two people to approve this ideally.

This also should update our readme and add the first of my code reviews. I still have more documentation to write, so I will need one more pull request this evening, but I wanted people to get this branch sooner than later.

### CODE REVIEW

File: Sprintfinity3902/Code Reviews/Kyle-ReadabilityReviewBoomerang.txt [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+seperate. 
+
+I think of all the methods, moveItem is the hardest to read. Effectively, there
+are three states the boomerang can be while its moving. Going out, Returning,
+and being caught. The if logic seperates those three states, but it is not
+clear upon reading which state is which. For someone who isn't involved with
+the code, it would hard to process what kind of logic is going on within
+those if statements.
+
+If you needed to add a new state to the logic, this would just further the
```
Comment (bradknez) [inline Sprintfinity3902/Code Reviews/Kyle-ReadabilityReviewBoomerang.txt:29]: Good suggestion! Definitely something to look at when we're doing item collision in Sprint 3.

**Review by bradknez — APPROVED** [review]
Tested, all works well and looks good. Good notes on the Boomerang class, definitely can help me improve it in the future. Thanks!

**Review by caseyvanc — APPROVED** [review]
Looks good!

**Review by Bailey-Ardrey — APPROVED** [review]
Looks good! Most of these changes we discussed together.

**Pull request merged**